src/components/ViewPort/ViewPort.tsx

```diff
@@ -19,7 +20,7 @@ export default class ViewPort extends React.PureComponent<Props> {
     const { isPreviewing, isReadOnly, type } = this.props
     return (
       <>
-        <Preview isReadOnly={isReadOnly === true} type={type || 'project'} />
+        <Preview isReadOnly={isReadOnly === true} type={type || PreviewType.PROJECT} />
```

Comment:
not PR related but, is the `=== true` necessary here?

Reply:
done 👍